src/cljs/ui/component/app_container.cljs

```diff
@@ -7,73 +7,73 @@
+                      [:span.badge {:class (cond
+                                             (= status "ERROR") "error"
+                                             new?               "new")} badge-text])]
+       [:div.sub-name {:style {:grid-area "sub-name"}} (type->label of-type)]
 
-  (let [menu-open?  (reagent/atom false)
-        active-page (re-frame/subscribe [::router.subs/active-page])]
-    (fn [{:keys [id readable-name of-type status new?]}]
-      (let [badge-text (cond
-                         (= status "ERROR") "Error"
-                         new?               "New")]
-        [:div.completed-menu-item.clickable {:on-click #(dispatch-n [[:router/navigate :route/analysis-results nil {:id id :tab "results"}]
-                                                                     (when new?
-                                                                       [:graphql/query {:query
-                                                                                        "mutation TouchAnalysisMutation($analysisId: ID!) {
-                                                                                 touchAnalysis(id: $analysisId) {
-                                                                                   id
-                                                                                   isNew
-                                                                                 }
-                                                                               }"
-                                                                                        :variables {:analysisId id}}])])}
-         [:div.readable-name {:style {:grid-area "readable-name"}} (or readable-name "Unknown")]
-         [:div.badges (when badge-text
-                        [:span.badge {:class (cond
-                                               (= status "ERROR") "error"
-                                               new? "new")} badge-text])]
-         [:div.sub-name {:style {:grid-area "sub-name"}} (type->label of-type)]
-         [:div {:style    {:grid-area "menu"}
-                :on-click #(swap! menu-open? not)}
-          [:img {:src "icons/icn_kebab_menu.svg"}]]
-         (when @menu-open?
-           [:ul.menu
-            [:li {:on-click #()} "Edit"]
-            [:li {:on-click #()} "Load different file"]
-            [:li {:on-click #()} "Copy settings"]
-            [:li {:on-click (fn [event]
-                              (let [{active-route-name :name query :query} @active-page]
-                                (.stopPropagation event)
-
-                                ;; if on results page for this analysis we need to nav back to home
-                                (when (and (= :route/analysis-results  active-route-name)
-                                           (= id (:id query)))
-                                  (>evt [:router/navigate :route/home]))
-
+       [icon-button {:style         {:grid-area "menu"}
```

Comment:
I think this should be a `shared.components/button` which already supports an icon since we are moving out of material-ui because of styling issues

Reply:
Done